CODEOWNERS

```diff
@@ -687,7 +687,7 @@
 /samples/subsys/logging/                  @nordic-krch @jakub-uC
 /samples/subsys/logging/syst/             @dcpleung
 /samples/subsys/shell/                    @jakub-uC @nordic-krch
-/samples/subsys/mgmt/mcumgr/smp_svr/      @aunsbjerg @nvlsianpu
+/samples/subsys/mgmt/mcumgr/              @de-nordic @nordicjm
```

Comment:
@aunsbjerg

Reply:
Note that the reason we removed @aunsbjerg is that he is already present in the specific file he maintains.

Reply:
Ok, just mention this in the commit message

Reply:
Done, thanks!